src/classes/RecipeRepository.js

```diff
@@ -17,7 +17,7 @@ class RecipeRepository {
   }
   filterName(name) {
     return this.recipes.filter((recipe) => {
-      return recipe.name === name;
+      return recipe.name.toLowerCase().includes(name)
```

Comment:
Great call changing the name from deeply equals to using the includes method and to lower case method to better the user experience!